### PR TITLE
Add pose conditioning images for validation

### PR DESCRIPTION
**Pull request by jfozard**
Small changes to allow pose validation images to be supplied to training code.

### CODE REVIEW

**Comment by jfozard** [comment]
Retain relevant parts of input convolution kernel weights, set remaining part to zero.

File: src/train_controlnet_flax.py [hunk 1]
```diff
@@ -854,6 +894,8 @@ def compute_snr(timesteps):
         return snr
 
     def train_step(state, unet_params, text_encoder_params, vae_params, batch, train_rng):
+
```
Comment (matallanas) [inline src/train_controlnet_flax.py:912]: Remove this space

File: src/train_controlnet_flax.py [hunk 2]
```diff
@@ -972,6 +1014,9 @@ def cumul_grad_step(grad_idx, loss_grad_rng):
             )
             loss, grad = jax.tree_map(lambda x: x / args.gradient_accumulation_steps, (loss, grad))
 
+
+
```
Comment (matallanas) [inline src/train_controlnet_flax.py]: Remove extra space

File: src/train_controlnet_flax.py [hunk 3]
```diff
@@ -1065,6 +1110,8 @@ def l2(xs):
                 train_metric["loss"].block_until_ready()
                 jax.profiler.stop_trace()
 
+
```
Comment (matallanas) [inline src/train_controlnet_flax.py]: Remove extra space

**Review by matallanas — APPROVED** [review]
Everything seems good, only some formats.

**Pull request merged**